concent_api/core/tests/test_unit_validation.py

```diff
@@ -441,3 +447,212 @@ def test_that_unsupported_format_raises_concent_validation_error(self):  # pylin
         with pytest.raises(ConcentValidationError) as exception_wrapper:
             validate_blender_output_format(unsupported_format)
         assert_that(exception_wrapper.value.error_code).is_equal_to(ErrorCode.MESSAGE_VALUE_NOT_ALLOWED)
+
+
+class TestValidateSceneResolution:
+
+    @pytest.mark.parametrize(
+        'resolution', [
+            '400x400',
+            [0, 0],
+            [100, 0],
+            [0, 100],
+            [-100, 100],
+            [100, -100],
+            ['100', 100],
+            [],
+            ['400x800'],
+            [71830.23, 1000],
+        ]
+    )  # pylint: disable=no-self-use
+    def test_that_invalid_resolution_value_raise_exception(self, resolution):
+        with pytest.raises(ConcentValidationError) as exception_wrapper:
+            validate_resolution(resolution)
+        assert_that(exception_wrapper.value.error_code).is_equal_to(ErrorCode.MESSAGE_INVALID)
```

Comment:
I was thinking about it, but until Concent's `error_messages` is not standardized and all error_messages are string then it won't be effective to check it because it might change. Also if we want to check it this test will be illegible:) The most important thing in this test is to check if all wrong values will raise exception. If we want to check that all validation has been tested we could run tests with coverage and then we can see that everything has been tested :)